src/Mod/Path/PathScripts/post/heidenhain_post.py

```diff
@@ -800,7 +800,7 @@ def HEIDEN_PolarArc(pol_cc_X, pol_cc_Y, pol_X, pol_Y, pol_Z, pol_Axis, pol_Incr)
         pol_Angle = 90 - pol_Angle
     # with Y axis zero have the Z+ direction
     elif pol_Axis == "Y":
-        pol_Angle = pol_Angle
+        pol_Angle = 0 + pol_Angle
```

Comment:
why not use `pass` instead? https://docs.python.org/3/tutorial/controlflow.html#pass-statements
```suggestion
        pass
```

Reply:
I'm not a programmer, i know only simple code.
I think your suggestion could work well